Dockerfile

```diff
@@ -0,0 +1,37 @@
+ARG buildImage=node:16.13.0-alpine
+ARG appPath=/usr/src/app
+
+# BE
+FROM $buildImage as be-build-stage
+WORKDIR $appPath/be
+COPY ./be/package*.json ./
+RUN npm ci
+COPY ./be .
+RUN npm run build #TODO After TM-94 merge: generate:build
+RUN npm prune
+
+# FE
+FROM $buildImage as fe-build-stage
+WORKDIR $appPath/fe
+COPY ./fe/package*.json ./
+RUN npm ci
+COPY ./fe/ .
+RUN npm run build # TODO generate:build, java not found error
+RUN npm prune
+
+# Merge BE + FE
+FROM $buildImage as production-stage
+WORKDIR $appPath/
+COPY --from=be-build-stage ./be/dist ./dist
+COPY --from=be-build-stage ./be/node_modules ./node_modules
```

Comment:
Are the `node_modules` still required in the final image?

If yes - can we make sure that we don't include the dev-dependencies?

Reply:
Yes is needed, `npm prune` removes all unneeded dependencies including dev ones, I added the `NODE_ENV=production`as it is required